### ISSUE TITLE
Verify that the SCD40 works as a drop-in replacement for the SCD41

### ISSUE THREAD

**Issue by danwahl** [danwahl/adanet-co2-monitor#8]
The [Low Power Operation document](https://sensirion.com/media/documents/077BC86F/62BF01B9/CD_AN_SCD4x_Low_Power_Operation_D1.pdf) says:

> Power Cycled Single Shot Operation (SCD41 only)

But the firmware is not currently using single-shot operation, so I don't think this applies.

 Still, worth verifying before purchasing SCD40s in bulk!

**Comment by danwahl** [comment]
I verified this, but still purchased the SCD41 for the first build round (since it was in stock). Given the functionality currently being used, I don't think it matters which of the SCD4x sensors is used. Keeping this open as a reminder to adjust the README accordingly.

**Comment by danwahl** [comment]
I confirmed that the SCD40 works using the periodic measurement mode, same as the SCD41 (which is expected).

I then tried to use the "Power Cycled Single Shot Operation" as described in the above document (which required modifying the [arduino-i2c-scd4x](https://github.com/danwahl/arduino-i2c-scd4x/tree/idle-single-shot) library) but was getting suspicious temperature/humidity readings.

I'm not 100% sure that using the sensor in periodic measurement mode for two samples is a valid way to do low-power operation, and so I updated the README to recommend the SCD41 for now (so that a firmware fix could in theory resolve any issues).
